.Lib9c.Tests/Action/StakeTest.cs

```diff
@@ -132,6 +134,19 @@ public void Execute_Throws_WhenCancelOrUpdateWhileLockup()
             }));
         }
 
+        [Fact]
+        public void Execute_Throws_WhenStakeLessThanMinimumRequiredGold()
+        {
+            Assert.Equal(50, _tableSheets.StakeRegularRewardSheet.Min(x => x.Value.RequiredGold));
```

Comment:
I think checking the requiredGold is greater than 10 would be enough in this case.